sympy/concrete/tests/test_sums_products.py

```diff
@@ -925,24 +925,22 @@ def test_is_convergent():
     assert Sum((-1)**n, (n, 1, oo)).is_convergent() is S.false
     assert Sum(log(1/n), (n, 2, oo)).is_convergent() is S.false
 
-    # root test --
+    # Cauchy's root test --
     assert Sum((-12)**n/n, (n, 1, oo)).is_convergent() is S.false
     assert Sum(2**n/factorial(n), (n, 1, oo)).is_convergent() is S.true
 
-    # integral test --
+    # integral test -- to be added
 
     # p-series test --
     assert Sum(1/(n**2 + 1), (n, 1, oo)).is_convergent() is S.true
     assert Sum(1/n**(S(6)/5), (n, 1, oo)).is_convergent() is S.true
     assert Sum(2/(n*sqrt(n - 1)), (n, 2, oo)).is_convergent() is S.true
 
     # comparison test --
-    assert Sum(1/(n + log(n)), (n, 1, oo)).is_convergent() is S.false
```

Comment:
why this removal?

Reply:
refer line 982

Reply:
@Upabjojr, a `XFAIL` function has been created for the same.